fizzbuzz_extended/spec/fizzbuzz_extended_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'fizzbuzz_extended'
+require 'minitest/autorun'
+
+class FizzBuzzExtendedTest < Minitest::Test
+  def setup
+    @h = { 7 => 'Siiv'}
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.